test/spec/core/CanvasSpec.js

```diff
@@ -1975,6 +2007,190 @@ describe('Canvas', function() {
 
   });
 
+
+  describe('planes', function() {
+
+    beforeEach(function() {
+      container = TestContainer.get(this);
+    });
+    beforeEach(createDiagram({ canvas: { width: 300, height: 300 } }));
+
+    describe('#getActivePlane', function() {
+      it('should return a default plane', inject(function(canvas) {
+
+        // when
+        var plane = canvas.getActivePlane();
+
+        // then
+        expect(plane).to.exist;
+        expect(plane.name).to.equal('base');
```

Comment:
Cf. above: What about root element?

What is our preferred and supported way to import.